src/cljs/rems/actions.cljs

```diff
@@ -12,61 +14,101 @@
 (rf/reg-event-fx
  ::enter-page
  (fn [{:keys [db]} _]
-   {:db (dissoc db ::todo-applications ::handled-applications)
+   {:db (-> db
+            (dissoc ::todo-applications
+                    ::handled-applications)
+            (assoc ::initializing-todo-applications? true
+                   ::initializing-handled-applications? true))
```

Comment:
should you set `:fethcing-todo-applications? false` here? or is it left out since there might actually be a fetch still in progress?